src/app/components/utility/input/input.component.ts

```diff
@@ -218,6 +218,10 @@ export class InputComponent implements OnInit {
        this.isValid = this.globalService.validatePassword(e);
        this.isValid ? this.message = '' : this.message = 'Password minimum 8 characters';
     }
+    if (this.name === 'bannedEmailIds' || this.name === 'filterByTeamName') {
```

Comment:
Is this change related to #163?

Reply:
yeah actually this was build on top of that, therefore it is showing here. Ideally you should review first that PR.

Reply:
OKay, cool.

Reply:
@lunayach Removed it from here for now, it will get added in `ban participant` PR